lxd/resource_lxd_container.go

```diff
@@ -85,6 +90,44 @@ func resourceLxdContainer() *schema.Resource {
 				ForceNew: false,
 			},
 
+			"file": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"content": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"target_file": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"uid": &schema.Schema{
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+
+						"gid": &schema.Schema{
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+
+						"mode": &schema.Schema{
+							Type:     schema.TypeString,
```

Comment:
What is the reason for making this a string, rather than Int ?
It seems later on, you expect octal file mode only, so why allow string input?